notary-commons/src/main/kotlin/config/Configs.kt

```diff
@@ -71,8 +71,6 @@ interface EthereumConfig {
  * Bitcoin configurations
  */
 interface BitcoinConfig {
-    //Path of wallet file
-    val walletPath: String
```

Comment:
why not to introduce 2 wallets here?

Reply:
Some services don't need two wallets. Like `btc-address-generation`